SmartDeviceLink/SDLArtwork.m

```diff
@@ -105,7 +107,8 @@ + (NSString *)sdl_md5HashFromNSData:(NSData *)data {
     unsigned char hash[CC_MD5_DIGEST_LENGTH];
     CC_MD5([data bytes], (CC_LONG)[data length], hash);
     NSMutableString *formattedHash = [NSMutableString stringWithCapacity:CC_MD5_DIGEST_LENGTH * 2];
-    for (int i = 0; i < CC_MD5_DIGEST_LENGTH; i += 1) {
+    // HAX: To shorten the string to 16 characters, the loop has been shortened to 8 fom 16.
+    for (int i = 0; i < 8; i += 1) {
```

Comment:
Should this be `for (int i = 0; i < CC_MD5_DIGEST_LENGTH / 2; i += 1) {`?

Reply:
Fixed